homeassistant/components/rtsp_to_webrtc/strings.json

```diff
@@ -19,7 +19,9 @@
       "server_unreachable": "Unable to communicate with RTSPtoWebRTC server. Check logs for more information."
     },
     "abort": {
-      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "server_failure": "RTSPtoWebRTC server returned an error. Check logs for more information.",
```

Comment:
These are copied from "error" above which I assume is the right thing to do.  I don't have an easy way to manually test hassio discovery, so looking for reviewer input on this.